src/components/Weather.js

```diff
@@ -0,0 +1,10 @@
+export const Weather = () => {
+  return (
+    <>

```

Comment:
here as well. if the child inside the fragment is one parent element, you can omit the empty fragments.

Reply:
yes, good point! , but this is just dummy data in the component, will work on it & remove empty fragments in the upcoming PRs.